src/utilities/buildClientSchema.js

```diff
@@ -232,19 +232,28 @@ export function buildClientSchema(
     });
   }
 
-  function buildObjectDef(
-    objectIntrospection: IntrospectionObjectType,
-  ): GraphQLObjectType {
-    if (!objectIntrospection.interfaces) {
+  function buildImplementationsList(
+    implementingIntrospection:
+      | IntrospectionObjectType
+      | IntrospectionInterfaceType,
+  ) {
+    if (!implementingIntrospection.interfaces) {
```

Comment:
We need to have legacy support for servers that would return `null` as `interfaces` value.
Fixed in https://github.com/mike-marcacci/graphql-js/pull/1/commits/07d8a2261f654768143a921507a1c96ce1cf645a